tests/crud/test_hooks.py

```diff
@@ -0,0 +1,182 @@
+import json
+from unittest import TestCase
+
+from piccolo.columns import Integer, Varchar
+from piccolo.columns.readable import Readable
+from piccolo.table import Table
+from starlette.testclient import TestClient
+
+from piccolo_api.crud.endpoints import PiccoloCRUD
+from piccolo_api.crud.hooks import Hook, HookType
+
+
+class Movie(Table):
+    name = Varchar(length=100, required=True)
+    rating = Integer()
+
+    @classmethod
+    def get_readable(cls):
+        return Readable(template="%s", columns=[cls.name])
+
+
+async def set_movie_rating_10(row: Movie):
+    row.rating = 10  # type: ignore
+    return row
+
+
+async def set_movie_rating_20(row: Movie):
+    row.rating = 20  # type: ignore
+    return row
```

Comment:
You can use this `async def set_movie_rating_20(row: t.Any = Movie):` or `async def set_movie_rating_20(row):` to remove `# type: ignore`

Reply:
Another way around this is to do `row['rating'] = 20`